circle.yml

```diff
@@ -28,9 +30,11 @@ test:
     - ./configure -i
   override:
     - env CXXFLAGS="-Werror" make test
+    - valgrind --leak-check=full --show-leak-kinds=definite,indirect,possible --error-exitcode=1 ./bin/test-libdrafter 2> memcheck.log
```

Comment:
Do you think it would make sense to move this to the makefile?

```shell
$ make memory-test
```

I can see this useful for running locally without remembering all of these args.